packer/mongo20220708.sh

```diff
@@ -9,31 +9,33 @@ yum -q makecache -y --disablerepo='*' --enablerepo='newrelic-infra'
 yum -y install newrelic-infra
 
 # Setup PostGres for Mongo Migraton
-yum -y install postgresql-server
+rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+yum install -y postgresql10-server
 
-postgresql-setup initdb
+#postgresql-setup initdb
+/usr/pgsql-10/bin/postgresql-10-setup initdb
 echo "
 # TYPE    DATABASE    USER    ADDRESS         METHODS
 local     all         all                     peer
 host      all         all     127.0.0.1/32    password
 host      all         all     ::1/128         password
-" > /var/lib/pgsql/data/pg_hba.conf
-systemctl start postgresql
-systemctl enable postgresql
+" > /var/lib/pgsql/10/data/pg_hba.conf
+systemctl enable postgresql-10
```

Comment:
I think we need to get onto a newer Postgres version soon

Reply:
Lets talk about it at Dev Sync